### ISSUE TITLE
[calendar header] Next/Prev buttons are not keyboard accessible.

### ISSUE THREAD

**Issue by allen138** [SAP/ui5-webcomponents#5787]
### **Bug Description**
Hi team,
The next/prev buttons in the calendar header are not able to receive focus. They are interactive elements, but they are inaccessible for a keyboard only user.

![image](https://user-images.githubusercontent.com/38701667/188736250-0bca7946-c1ed-4519-86eb-d9e5c75836e3.png)

Workaround: Use month/year picker to navigate through various months and years. Also when focus is on the day, user can use the arrow keys navigating all the way down or up will render the next/previous month.

### **Expected Behavior**
The next/prev buttons in the calendar header should be accessible for a keyboard only user.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/main/playground/components/Calendar/
2. Tab through the calendar. Notice you are able to focus on the month, year and day buttons/element but you cannot focus on the next/prev buttons in the header.

### **Context**
- UI5 Web Components version: main
- OS/Platform: all
- Browser: all
- Affected component: ui5-calendar

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SF
- Business impact: Medium

**Comment by LidiyaGeorgieva** [comment]
Hello @SAP/ui5-webcomponents-topic-b please take a look. I see that the [openui5 calendar](https://sapui5untested.int.sap.eu2.hana.ondemand.com/#/entity/sap.ui.unified.Calendar/sample/sap.ui.unified.sample.CalendarSingleDaySelection) has the same behavior.

**Comment by allen138** [comment]
Hi @SAP/ui5-webcomponents-topic-b, I also tested openui5 calendar, and date pickets and confirm that it is the same behavior as @LidiyaGeorgieva has mentioned above.

May I get some justification as to why these two elements are not focusable, so I can relay that back to our ACC team and then we can close this ticket.

Thanks,
Allen

**Comment by NHristov-sap** [comment]
Hi @allen138 ,

As it is described in the specification of the Calendar, the mentioned arrows are not supposed to be used with keyboard interaction. Their purpose is to be used with mouse. Here's some parts of the keyboard handling specification:

[TAB] Forward navigation:
• On enter, move focus to calendar
• Afterwards, move focus to month button
• Then, move focus to year button
• On leave, move focus to the next interactive element in the tab chain after the calendar.

[SHIFT]+[TAB] Backward navigation:
• On enter, move focus to year button (OK)
• Afterwards, move focus to month button (OK)
• Then, move focus to calendar (OK)
On leave, move focus to the previous interactive element in the tab chain before the calendar. (OK)

There are dedicated keyboard shortcust for navigating between months:

[PAGE UP] Moves focus to the same item in the previous page (same day, previous month). Decouples focus from selection.
[PAGE DOWN] Moves focus to the same item in the next page (same day, next month). Decouples focus from selection.

As it is seen, the Calendar works according to the Keyboard handling specifications, that's why I will close this issue.

Best Regards,
Nikolay Hristov
SAPUI5
SAP Development Support

**Comment by allen138** [comment]
Thank you @NHristov-sap for the explanation
